dev/examples/ThreeBox.tsx

```diff
@@ -43,11 +61,9 @@ function Box(props) {
 
 export const App = () => {
     const [isHovered, setHover] = useState(false)
+    console.log({ isHovered })
     return (
-        <MotionConfig
-            isStatic
-            transition={{ type: "spring", bounce: 0, duration: 0.7 }}
-        >
+        <MotionConfig transition={{ type: "spring", bounce: 0, duration: 0.7 }}>
```

Comment:
Why is the `isStatic` removed?

Reply:
I'm not sure why it was there tbh, probably mucking about with passing context via `MotionCanvas`. But it was disabling the animations in the sandbox.